src/main/resources/application.yaml

```diff
@@ -107,7 +107,7 @@ ftp:
     - service: send_letter_tests
       folder: BULKPRINT
     - service: nfdiv_case_api
-      folder: DIVORCE
+      folder: ${NFD_SERVICE_FOLDER:DIVORCE}
```

Comment:
Should not it be
${NFD_SERVICE_FOLDER:NFDIVORCE}
?

Reply:
defaulting it to DIVORCE to avoid issues in production